src/runtime-rs/crates/resource/src/manager_inner.rs

```diff
@@ -232,6 +238,20 @@ impl ResourceManagerInner {
             .await
     }
 
+    pub async fn handle_sandbox_bindmounts(&self, setup: bool) -> Result<()> {
```

Comment:
How about seperating this function into `setup_sandbox_bindmounts` and `cleanup_sandbox_bindmounts`, and abstract the `get_sandbox_bindmounts` into one function ? Using `setup` parameter to control setup or cleanup is a little wierd for me.